transactions/signer_revoke.cdc

```diff
@@ -28,11 +29,10 @@ transaction(treasuryAddr: Address, actionUUID: UInt64, message: String, keyIds:
 
   }
   execute {
-    self.isValid = self.action.signerRevokeApproval(messageSignaturePayload: self.messageSignaturePayload)
+    self.isValid = self.manager.signerRejectAction(actionUUID: actionUUID, messageSignaturePayload: self.messageSignaturePayload)
   }
 
   post {
     self.isValid == true: "Unable to revoke approval: invalid message or signature"
-    self.action.accountsVerified[self.messageSignaturePayload.signingAddr] == false: "Error: tx completed but signer approval not revoked"
```

Comment:
maybe we should leave this check too like for approve

Reply:
the reason I removed it is because the case where the signer revokes it, and the action gets destroyed.  in this case, the transaction always fails on this `post` condition because `self.action` no longer exists.

i couldn't figure out any way to check if `self.action` has been destroyed. and something like `self.action && self.action.accountsVerified[addr] == false` also fails, because it is still trying to evaluate `self.action.accountsVerified`

I'll ask in AZ slack to confirm if its possible to check whether a resource you have a reference to has been destroyed.

Reply:
it is not possible to check if a resource is destroyed in Cadence, according to Bastian.

we could add a separate return value which indicates that the resource has been destroyed, but i don't think its necessary.